src/main/scala/ch/epfl/bluebrain/nexus/kg/marshallers/ExceptionHandling.scala

```diff
@@ -23,7 +25,8 @@ object ExceptionHandling {
     */
   final def apply(): ExceptionHandler =
     ExceptionHandler {
-      case rej: Rejection => complete(rej)
+      case rej: Rejection                   => complete(rej)
+      case ElasticClientError(status, body) => complete(status -> parse(body))
```

Comment:
You can also use the more generic case
```scala
case f: ElasticFailure => complete(status -> parse(f.body))
```

Reply:
Make it return a `Rejection`: 
```scala
complete(parse.left.map[Rejection](Unexpected("ElasticSearch response '$body'")))
```

Reply:
@umbreak We probably don't want to forward ES server errors, so I don't think using the generic class/trait makes sense here.